source/aerobulk/flux.py

```diff
@@ -9,29 +13,209 @@ def _check_algo(algo, valids):
         raise ValueError(f"Algorithm {algo} not valid. Choose from {valids}.")
 
 
-def flux_noskin(
-    sst, t_zt, hum_zt, u_zu, v_zu, slp=101000.0, algo="coare3p0", zt=10, zu=2, niter=1
+def noskin_np(
+    sst,
+    t_zt,
+    hum_zt,
+    u_zu,
+    v_zu,
+    slp,
+    algo,
+    zt,
+    zu,
+    niter,
 ):
     """Python wrapper for aerobulk without skin correction.
     !ATTENTION If input not provided in correct units, will crash.
```

Comment:
This is a very vague warning. I would prefer that we actually listed the allowed value ranges under each input in the docstring.

Reply:
I also don't think we should describe this error in terms of "correct units". The function expects the values to be expressed in certain units, AND the function will error if the values given are outside some (completely arbitrary) range.

Reply:
Also you should be consistent with using the Warnings RST block.